modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/checkbox/CheckboxDDMFormFieldTemplateContextContributor.java

```diff
@@ -84,22 +90,24 @@
 				"tooltip")
 		).put(
 			"value",
-			GetterUtil.getBoolean(ddmFormFieldRenderingContext.getValue())
+			GetterUtil.getBoolean(
+				getValue(ddmFormFieldRenderingContext.getValue()))
 		).build();
 	}
 
-	protected String getPredefinedValue(
-		DDMFormField ddmFormField,
-		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
-
-		LocalizedValue predefinedValue = ddmFormField.getPredefinedValue();
+	protected String getValue(String valueString) {
```

Comment:
@marcelabc, from what I could see, the both methods `getValue` has the same logic, am I right?

If so, do you think we can move this method to a single place, like a util class, for instance, and reuse it in both places?